### ISSUE TITLE
Doesn't work on miui 13.0.5 (android 12)

### ISSUE THREAD

**Issue by Wjxfi** [bmx666/android-appcachecleaner#92]
it goes to the settings of the first application in the list and then nothing happens. Permissions are granted

**Comment by bmx666** [comment]
Hi @Wjxfi  Is the same as here #65  ?

**Comment by bmx666** [comment]
Hi @Wjxfi 

> But you can't close an issue knowing that your app doesn't work on all xiaomi phones. 

It was closed not by me.

> Fix it.

I don't owe you anything, I made it open to the rest of the world and put it on F-Droid and Google Play not to make a lot of money. I made this for people who are fed up with "delay" and paying for simple actions. I don't  have and I don't know anyone in my environment who has Xiaomi phones.

> I looked at this section. It's too complicated. Do it automatically. No one is going to bother with that for one application.

Don't use Cache Cleaner, use other app. There are a lot of alternatives that track, collect and sell your personal data. It's up to you.
